source/Octopus.Server.Client/Model/VariableType.cs

```diff
@@ -7,6 +7,7 @@ public enum VariableType
         Certificate,
         AmazonWebServicesAccount,
         AzureAccount,
-        WorkerPool
+        WorkerPool,
+        UsernamePassword
```

Comment:
This type is a slightly different convention to the other accounts, which all have the `Account` suffix.